workflow/executor/resource.go

```diff
@@ -311,14 +311,37 @@ func (we *WorkflowExecutor) SaveResourceParameters(ctx context.Context, resource
 		}
 		output := string(out)
 		if param.ValueFrom.JQFilter != "" {
```

Comment:
This looks really nice! Are the some tests to verify this logic? Maybe we could add one if not some old?